### ISSUE TITLE
Is org-ql-view redundant since emacs has bookmarks?

### ISSUE THREAD

**Issue by codygman** [alphapapa/org-ql#291]
Or perhaps saving a view should create a bookmark, or a configuration option should be added for this?

Maybe removing org-ql-view and encouraging users to use bookmarks would be a good simplification?

Or maybe things are fine as is, I just noticed I've just been using both and caused myself some personal confusion.

**Comment by alphapapa** [comment]
Yes...no...maybe...?  I added `org-ql-views` before I added Emacs bookmark support.  But org-ql-views still provide a way to view *only* org-ql views as opposed to other bookmarks, and there's also the sidebar command to show them in a window for easy switching.  So...someday I'll probably revisit all that, but it's not a priority compared to other plans.

**Comment by ParetoOptimalDev** [comment]
Understandable. Feel free to close if you want, I remember you like to keep some issues open.

**Comment by alphapapa** [comment]
I'll probably mark `org-ql-view`'s bespoke view-saving system as obsolete in v0.9 and recommend that users use bookmarks instead.  At the same time I'll probably adjust the commands to also work with bookmarks so that users can transition (i.e. a command will offer bookmarked Org QL views without showing other kinds of bookmarks).  Then in v0.10 I'll probably remove support for the bespoke saved views.
